crates/api_common/src/comment.rs

```diff
@@ -69,7 +69,7 @@ pub struct CreateCommentLike {
   pub auth: Sensitive<String>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, Default)]
```

Comment:
Deriving default is useful for client development, when you just want to specify one or two params, and not put all the None values explicitly.

Reply:
I'd probably prefer all of them to derive default, and just make sure the string cases are done by convention. API clients should be will likely find those issues during development anyway.